### PR TITLE
fix: refactor driveteam buttons 

### PR DESCRIPTION
**Pull request by sanashah007**

### CODE REVIEW

**Comment by rutmanz** [comment]
Ball eject buttons will be added by #77

File: src/main/java/org/team1540/robot2022/RobotContainer.java [hunk 1]
```diff
@@ -89,53 +89,38 @@ private void configureButtonBindings() {
...
-        // coop:button(B,Toggle intake fold [press],pilot)
-        new JoystickButton(driverController, Button.kB.value)
-                .whenPressed(new InstantCommand(() -> intake.setFold(!intake.getFold())));
-
         // Copilot
 
-        // coop:button(B,Toggle intake fold [press],copilot)
-        new JoystickButton(copilotController, Button.kB.value)
+        // coop:button(RBumper,Toggle intake fold [press],copilot) KEEPING
+        new JoystickButton(copilotController, Button.kRightBumper.value)
```
Comment (rutmanz) [inline src/main/java/org/team1540/robot2022/RobotContainer.java:95]: You're copilot, but I'm not sure it makes much sense for:
`Left bumper` to be manual intake,
`B` be manual reverse intake, and
`Right bumper` to be folding the intake

I think this was fine before

File: src/main/java/org/team1540/robot2022/RobotContainer.java [hunk 2]
```diff
@@ -89,53 +89,38 @@ private void configureButtonBindings() {
...
 
-        // coop:button(LBumper,Manual intake [hold],copilot)
+        // coop:button(LBumper,Manual intake [hold],copilot) KEEPING
         new JoystickButton(copilotController, Button.kLeftBumper.value)
                 .whileHeld(new IntakeSpinCommand(intake, indexer, Constants.IntakeConstants.SPEED));
-        // coop:button(RBumper,Manual reverse intake [hold],copilot)
-        new JoystickButton(copilotController, Button.kRightBumper.value)
+                
+        // coop:button(B,Manual outtake [hold],copilot) CORRECT
+        new JoystickButton(copilotController, Button.kB.value)
```
Comment (rutmanz) [inline src/main/java/org/team1540/robot2022/RobotContainer.java:103]: We probably don't want anything on ABXY to be held down, it's not in a very good position for that

File: src/main/java/org/team1540/robot2022/RobotContainer.java [hunk 3]
```diff
@@ -89,53 +89,38 @@ private void configureButtonBindings() {
...
                 .whileHeld(new IntakeSpinCommand(intake, -Constants.IntakeConstants.SPEED));
-
-        // coop:button(DPadRight,Run intake and indexer [press],copilot)
+        //coop:button(A, acquire ball, copilot) CORRECT
```
Comment (rutmanz) [inline src/main/java/org/team1540/robot2022/RobotContainer.java:105]:
```suggestion
        //coop:button(A, Run intake sequence [press], copilot)
```
"Acquire ball" makes me think of an instant event, not something that is continuous

File: src/main/java/org/team1540/robot2022/RobotContainer.java [hunk 4]
```diff
@@ -89,53 +89,38 @@ private void configureButtonBindings() {
...
+                
+        // coop:button(B,Manual outtake [hold],copilot) CORRECT
+        new JoystickButton(copilotController, Button.kB.value)
                 .whileHeld(new IntakeSpinCommand(intake, -Constants.IntakeConstants.SPEED));
-
-        // coop:button(DPadRight,Run intake and indexer [press],copilot)
+        //coop:button(A, acquire ball, copilot) CORRECT 
+        new JoystickButton(copilotController, Button.kA.value)
+                .cancelWhenPressed(indexerEjectCommand)
+                .whenPressed(intakeSequence);
```
Comment (rutmanz) [inline src/main/java/org/team1540/robot2022/RobotContainer.java:108]: Why do we have this and DPadRight running the same commands?

File: src/main/java/org/team1540/robot2022/RobotContainer.java [hunk 5]
```diff
@@ -89,53 +89,38 @@ private void configureButtonBindings() {
...
                 .whenPressed(intakeSequence);
-        // coop:button(DPadDown,Stop indexer and intake [press],copilot)
-        new POVButton(copilotController, 180) // D-pad down
+        // coop:button(DPadDown,Stop indexer and intake [press],copilot) KEEPING 
+        new POVButton(copilotController, 270) // D-pad left 
                 .cancelWhenPressed(indexerEjectCommand)
                 .cancelWhenPressed(intakeSequence);
-        // coop:button(DPadLeft,Eject balls from intake,copilot)
-        new POVButton(copilotController, 270) // D-pad left
+        // coop:button(DPadDown,climber solenoid false,copilot) CORRECT
```
Comment (rutmanz) [inline src/main/java/org/team1540/robot2022/RobotContainer.java:117]:
```suggestion
        // coop:button(DPadDown,Extend climber cylinders,copilot)
```
"climber solenoids false" doesn't have any verb in it, it's unclear what that means

File: src/main/java/org/team1540/robot2022/RobotContainer.java [hunk 6]
```diff
@@ -89,53 +89,38 @@ private void configureButtonBindings() {
...
+        new POVButton(copilotController, 180) // D-pad down
                 .whenPressed(new InstantCommand(() -> climber.setSolenoids(false)));
-        // coop:button(DPadUp,Toggle climber solenoids [press],copilot)
+        // coop:button(DPadUp, climber solenoids true,copilot) CORRECT
```
Comment (rutmanz) [inline src/main/java/org/team1540/robot2022/RobotContainer.java:120]:
```suggestion
        // coop:button(DPadUp, Retract climber cylinders,copilot)
```
Same as previous comment

**Review by rutmanz — CHANGES_REQUESTED** [review]
Do we still need the `KEEPING` and `CORRECT` comments for something or are those just from you making sure you had everything?

**Comment by natesales** [comment]
Sorry! #79 covers this and fixes most of the comments that @rutmanz raised.

**Pull request closed without merging**